home/modules/ROOT/pages/mobile.adoc

```diff
@@ -1,69 +1,99 @@
 
-[.content]
-Sample tutorials to build Mobile applications on the edge. +
-https://developer.couchbase.com/tutorials[Couchbase Mobile Tutorials]
+=== Edge/IoT Deployments (Optional)
+Components: Couchbase Lite + Edge Server + (Sync Gateway or App Services)
 
-++++
-</div>
-++++
+* For lightweight edge computing and IoT environments
+* Edge Server enables local data processing at the edge
+* Can sync with either on-premises Sync Gateway or cloud-based App Services
+* Ideal for scenarios requiring reduced latency and local data processing
+
+== Data Flow
+
+1. Couchbase Lite stores and syncs data locally on mobile devices, supporting offline operations.
+2. Sync Gateway (on-prem) or App Services (cloud) manage secure, bidirectional synchronization between devices and the backend database.
+3. For edge deployments, Edge Server provides local data processing and sync for IoT/edge devices.
+4. Capella offers a fully managed cloud database and sync solution
+
+== Mobile Development
+
+=== Couchbase Lite SDKs
+
+Build mobile apps with native SDKs for:
+
+* iOS/macOS: https://docs.couchbase.com/couchbase-lite/current/swift/quickstart.html[Swift], https://docs.couchbase.com/couchbase-lite/current/objc/quickstart.html[Objective-C]
+
+* Android: https://docs.couchbase.com/couchbase-lite/current/java/quickstart.html[Java], https://docs.couchbase.com/couchbase-lite/current/android/kotlin.html[Kotlin]
+
+* Cross-Platform: https://docs.couchbase.com/couchbase-lite/current/csharp/quickstart.html[.NET], https://docs.couchbase.com/couchbase-lite/current/c/quickstart.html[C]
+
+* JavaScript: https://docs.couchbase.com/couchbase-lite/current/javascript/ionic.html[Ionic], https://docs.couchbase.com/couchbase-lite/current/javascript/react.html[React Native]
+
+NOTE: JavaScript is community managed and not supported by Couchbase officially. 
+
+=== Sync Gateway Platforms
+Deploy Sync Gateway on:
+
+https://docs.couchbase.com/sync-gateway/current/get-started-install.html#install-for-linux[Linux]
+
+https://docs.couchbase.com/sync-gateway/current/get-started-install.html#install-for-windows[Windows]
+
+https://docs.couchbase.com/sync-gateway/current/get-started-install.html#install-for-mac-os[macOS]
+
+https://docs.couchbase.com/sync-gateway/current/deploy-cluster-to-kubernetes.html[Kubernetes]
+
+== Getting Started
+
+* For cloud development: https://docs.couchbase.com/home/cloud.html[Start with Capella] and https://docs.couchbase.com/cloud/app-services/index.html[App Services] for managed Sync. 
+* For on-premises: https://docs.couchbase.com/couchbase-lite/current/index.html[Install Couchbase Lite] and https://docs.couchbase.com/sync-gateway/current/introduction.html[configure Sync Gateway]
+* For Edge computing: https://docs.couchbase.com/couchbase-edge-server/current/introduction/intro.html[Deploy Edge Server] for local/IoT sync.
+
+== Learn More
+
+* https://developer.couchbase.com/tutorials/[Couchbase Mobile Tutorials]
```

Comment:
This hardcoded link is fine - it's external to docs, so we can't refer to it with the Antora syntax